### DISCUSSION TITLE
Craft 5 beta and project starting right now

### DISCUSSION THREAD

**Discussion by zsavajji** [nmaier95/craft-shopify-product-fetcher#92] [Q&A] [answered]
Hi! 
I am using Craft for some of my projects and I'm loving the experience!
I have a new project starting up soon which requires a Shopify integration, and I would like to start fresh with Craft5, since it'll be completed a bit after the release of Craft5.
Are there any plans to port the plugin soon, or at least to release a Craft5 compatible beta to test?

Thanks!

**Comment by sir-savien** [comment]
Bumping - i'd also would like to know this, so i can decide how to move forward with an upcoming project.

**Comment by lukeholder** [comment, marked as answer]
Shopify support Craft 5!
